assets/src/styles/pressbooks.scss

```diff
@@ -8,6 +8,12 @@
   transform: translateZ(0);
 }
 
+.toplevel_page_pb_import {
+  .wp-menu-image {
+    transform: rotate(180deg);
+  }
+}
+
```

Comment:
Right now I'm using the same icon from the "Export" menu, so I'm rotating it to avoid confusion. We could also pick a new icon from here https://developer.wordpress.org/resource/dashicons/#editor-ul